docs/intro/quickstart.md

```diff
@@ -0,0 +1,59 @@
+# Quick Start
+
+Get started with Krustlet in three easy steps:
+
+1. Boot a Kubernetes cluster
+2. Boot a Krustlet node
+3. Deploy your first application
+
+As Krustlet is under active development, this guide will help you set up a cluster suitable for evaluation, development,
+and testing purposes.
+
+## Step 1: Boot a Kubernetes Cluster
+
+There are many ways to boot up a Kubernetes cluster. You may choose to get up and running in cloud environments or
+locally on your laptop.
+
+If you have already created a Kubernetes cluster, check out the [system requirements](../topics/system-requirements.md)
+and then proceed to the next step to install Krustlet on your own Kubernetes cluster.
+
+### Cloud-based options
+
+For production use, you'll want to boot a Kubernetes cluster using a fully-managed service provided by a cloud vendor.
+
+- [Azure Kubernetes Service (AKS)](../howto/kubernetes-on-aks.md)
+
+### VM-based options
+
+For development and evaluation purposes, it may make sense to use a VM-based Kubernetes cluster for quick and easy setup
+and teardown.
+
+- [Kubernetes-in-Docker (KinD)](../howto/kubernetes-on-kind.md)
+
+## Step 2: Boot a Krustlet node
+
+Depending on whatever provider you chose in step 1, you now have a few options to boot and register Krustlet with your
+Kubernetes cluster.
+
+If you have your own Kubernetes cluster, you may want to follow the steps in the cloud-based option guides to determine
+how to set up Krustlet for your own infrastructure.
+
+### Cloud-based options
+
+For production use, you'll want to boot a new VM with Krustlet installed on the same cloud provider.
```

Comment:
I would just copy the sentence from the AKS doc that talks about all you need is a computing device (doesn't need to be a VM) with an IP address that the kubernetes control plane can talk to.